test/lib/generate/option-postprocess-test.ts

```diff
@@ -0,0 +1,60 @@
+import { generate } from '../../../lib/generator.js';
+import mockFs from 'mock-fs';
+import { dirname, resolve, relative } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { readFileSync } from 'node:fs';
+import { jest } from '@jest/globals';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const PATH_NODE_MODULES = resolve(__dirname, '..', '..', '..', 'node_modules');
+
+describe('generate (--postprocess)', function () {
```

Comment:
Since it's not a CLI option:
```suggestion
describe('generate (postprocess option)', function () {
```